test/acceptance/test/utils.go

```diff
@@ -298,6 +317,7 @@ func gitAddCommitPush(repoAbsolutePath string, appManifestFilePath string) {
 	command := exec.Command("sh", "-c", fmt.Sprintf(`
                             cp -r %s %s &&
                             cd %s &&
+							sleep 10s &&
```

Comment:
:worried: Why do we need to `sleep` after changing directories?

Reply:
You are right, actually this sleep should be after hub create, as it takes few secs after the call to create the repo on GitHub.

Reply:
Moved to sleep to the right function.